newdle/migrations/versions/20200914_1600_c161fdcfab19_.py

```diff
@@ -0,0 +1,31 @@
+"""Add notify column to db model
+
+Revision ID: c161fdcfab19
+Revises: 93a638b96375
+Create Date: 2020-09-14 16:00:33.501681
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'c161fdcfab19'
+down_revision = '93a638b96375'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        'newdles',
+        sa.Column('notify', sa.Boolean(), server_default='false', nullable=False),
```

Comment:
after adding the column (which applies the default to all existing rows), alter it to change the default to `true` (see the revision adding private for an example)